transportable-udfs-examples/transportable-udfs-example-udfs/build.gradle

```diff
@@ -7,18 +7,26 @@ buildscript {
 apply plugin: 'java'
 apply plugin: 'com.linkedin.transport.plugin'
 
+println 'Running build for Hive, Presto & Spark'
+transport.engines = ['hive', 'presto', 'spark']
```

Comment:
Agreed! but I am not sure if it is possible to control the jdk version from inside the plugin. I will explore the option you suggested below. If that works, it would be the best way forward.